### PR TITLE
Add STEP Main and Test networks

### PR DESCRIPTION
**Pull request by ongrid**

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/icons/step.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://QmVp9jyb3UFW71867yVtymmiRw7dPY4BTnsp3hEjr9tn8L",
```
[the remark this answers is not in the document]
Comment (ongrid) [reply _data/icons/step.json:3]: Pinata https://gateway.pinata.cloud/ipfs/QmVp9jyb3UFW71867yVtymmiRw7dPY4BTnsp3hEjr9tn8L

**Comment by ongrid** [comment]
Image is available on all IPFS gateways I tested, changed shortname to unique (CI was failing because of that)

**Comment by ongrid** [comment]
Here is the alternative link that confirms image available through another IPFS gateway https://ipfs.io/ipfs/QmVp9jyb3UFW71867yVtymmiRw7dPY4BTnsp3hEjr9tn8L Could you please check @ligi

**Comment by ongrid** [comment]
@ligi could you please take a look 👀

**Pull request closed without merging**